packages/pages/src/upgrade/pagesUpdater.test.ts

```diff
@@ -3,25 +3,54 @@ import { removeFetchImport } from "./pagesUpdater.js";
 import { Project } from "ts-morph";
 import path from "path";
 import fs from "fs";
+import typescript from "typescript";
 
 describe("test pages updater steps", () => {
-  it("remove fetch imports", () => {
+  it("removes fetch imports", () => {
+    const expectedBefore =
+      'import { fetch, tacos } from "@yext/pages/util";\n' +
+      'import fetch from "@fetch/fetcher";\n';
+    const expectedAfter =
+      'import { tacos } from "@yext/pages/util";\n' +
+      'import fetch from "@fetch/fetcher";\n';
+
+    // generate test file
     const testFilePath = "src/upgrade/test/src/templates/location.tsx";
-    const testProject = new Project();
+    const source = path.resolve("src/upgrade/test/src");
+    const testProject = new Project({
+      compilerOptions: {
+        jsx: typescript.JsxEmit.ReactJSX,
+        sourceRoot: source,
+      },
+    });
+    testProject.addSourceFilesAtPaths([
```

Comment:
You probably only want to add the one specific file you're testing against

Reply:
We actually don't need this addSourceFiles call here at all since we have one below. I'll remove it